From 65dafdec423660f324669b6f0c1f789068fcc276 Mon Sep 17 00:00:00 2001
From: Alexander Afanasyev <aa@cs.fiu.edu>
Date: Thu, 5 Jul 2018 01:26:52 -0400
Subject: [PATCH 2/2] build: Add additional boost libraries

For some reason, these libraries are needed while to cross-compile for
ARM64/Android.

Change-Id: I295d17a49547f5cd95b7a77cee95439f2236821e
---
 wscript | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/wscript b/wscript
index 8a5a2525..9e047d28 100644
--- a/wscript
+++ b/wscript
@@ -98,7 +98,7 @@ def configure(conf):
     conf.check_sqlite3(mandatory=True)
     conf.check_openssl(mandatory=True, atleast_version=0x1000200f) # 1.0.2
 
-    USED_BOOST_LIBS = ['system', 'filesystem', 'date_time', 'iostreams',
+    USED_BOOST_LIBS = ['system', 'atomic', 'regex', 'filesystem', 'date_time', 'iostreams',
                        'program_options', 'chrono', 'thread', 'log', 'log_setup']
 
     if conf.env['WITH_TESTS']:
-- 
2.20.1

